### PR TITLE
Resolve spelling mistakes

### PR DESCRIPTION
**Pull request by GovindaSharma**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

When this pull request is merged, it will fix spelling mistakes

## Additional information

List all affected URLs 

https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3579) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3579) before we can accept your contribution.<br/>

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by osrecio** [comment]
Hi @GovindaSharma thanks for your contribution.

One question you only change `caches` by `cache`. I don't see more changes. I'm right?

(To be clear is not a problem only change a word. Is only that I'm almost blind, but I do not see any more changes :) is to confirm it )

File: guides/v2.3/graphql/develop/resolvers.md [hunk 1]
```diff
@@ -7,6 +7,6 @@ A resolver performs GraphQL request processing. In general, it is responsible fo
...
+You can view an example of this can be found inside the [`\Magento\BundleGraphQl\Model\Resolver\BundleItemLinks`]({{ site.mage2300url }}app/code/Magento/BundleGraphQl/Model/Resolver/BundleItemLinks.php) resolver. This resolver takes each bundle option ID and its corresponding parent product ID and stores them in a collection's filter buffer (in this case, using the [`\Magento\BundleGraphQl\Model\Resolver\Links\Collection::addIdFilters()`]({{ site.mage2300url }}app/code/Magento/BundleGraphQl/Model/Resolver/Links/Collection.php#L62-L70) function). Each resolver then returns a callable that  invokes this collection. The collection cache the result of all link entities it fetched for all the option_id/parent_id combinations. This fetch only needs to occur once for the whole `BundleItemLink` list, and each resulting callable that is invoked for every link in the list returns an item from the collections cached result.
```
Comment (osrecio) [inline guides/v2.3/graphql/develop/resolvers.md]: Can you delete the space 
```suggestion
that invokes this collection.
```

**Comment by GovindaSharma** [comment]
@osrecio i have removed the extra space. and also since caches is not any word that's why i changed the caches to cache.

**Comment by keharper** [comment]
@GovindaSharma  The statement "The collection caches" was correct as written .  "Collection" is a collective noun and takes a 3rd-person singular verb--"caches" in this case.  

Please change "cache" back to "caches". However, you can help improve the documentation by changing this text

"You can view an example of this can be found inside"

to

"You can view an example inside"

**Comment by GovindaSharma** [comment]
@keharper i have updated the changes suggested by you and changed "cache" to "caches" back.
Thank you

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @GovindaSharma !

**Pull request merged**